app/web/src/features/communities/PageHeader.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { Meta, Story } from "@storybook/react";
+import community from "test/fixtures/community.json";
+
+import PageHeader from "./PageHeader";
+
+export default {
+  component: PageHeader,
+  title: "Communities/PageHeader",
+} as Meta;
+
+const Template: Story<any> = (args) => <PageHeader {...args} />;
```

Comment:
```suggestion
const Template: Story<PageHeaderProps> = (args) => <PageHeader {...args} />;
```
Instead of using `any` since the `args` here are the same as the prop type? I realised that means you'll need to extract the type into `PageHeaderProps` rather than inlining it in the component but that shouldn't be a big job!